service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApproveDraftOrdersControllerSubmittedTest.java

```diff
@@ -77,6 +83,9 @@
     @MockBean
     private CoreCaseDataService coreCaseDataService;
 
+    @MockBean
+    private SendLetterService sendLetters;
```

Comment:
ideally as integration test it should not mock this service but only low level services responsible for network communication, but can stay